app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/GetRecordsConnector.scala

```diff
@@ -61,9 +61,9 @@ class GetRecordsConnector @Inject() (
   def fetchRecords(
     eori: String,
     correlationId: String,
+    size: Int,
     lastUpdatedDate: Option[Instant] = None,
-    page: Option[Int] = None,
-    size: Option[Int] = None
+    page: Option[Int] = None
```

Comment:
I don't know enough about Scala standards, but I would prefer to keep page and size next to each other rather than split by lastUpdatedDate

Reply:
we need to pass mandatory fields before optionals, can we instead move page and size together and pass updated date as last parameter?

Reply:
That works for me :)

Reply:
done